Modules/AuctionManager/AuctionManager.lua

```diff
@@ -111,6 +111,15 @@ function AuctionManager:Initialize()
             -- width = "double",
             order = 34
         },
+        auctioning_raid_warn_countdown = {
```

Comment:
If you are adding countdown warning enable/disable I also think you should add enable/disable of all raid announcements (under single or multiple configs) to toggle: raid start/end warning, auction start/end warnings, loot award warning.